test/unit/spec/common/jobManager-Spec.js

```diff
@@ -577,24 +611,61 @@ define([
             ['cancel', 'retry'].forEach((action) => {
                 // cancelJob and retryJob: input is a single job ID
                 describe(`${action}Job`, () => {
+                    beforeEach(function () {
+                        const { model } = createModelWithBatchJobWithRetries();
+                        this.jobManagerInstance = new JobManager({
+                            model,
+                            bus: this.bus,
+                            devMode: true,
+                        });
+                    });
+
                     actionStatusMatrix[action].valid.forEach((status) => {
                         const jobId = JobsData.jobsByStatus[status][0].job_id;
                         it(`can ${action} a job in status ${status}`, function () {
                             spyOn(this.bus, 'emit');
-                            const result = this.jobManagerInstance[`${action}Job`](jobId);
+                            const result = this.jobManagerInstance[`${action}Job`](`${jobId}-v2`);
                             expect(result).toBeTrue();
                             expect(this.bus.emit).toHaveBeenCalled();
                             // check the args to bus.emit were correct
                             const callArgs = this.bus.emit.calls.allArgs();
                             const actionRequest = `request-job-${actionStatusMatrix[action].request}`;
-                            expect(callArgs[0]).toEqual([actionRequest, { jobIdList: [jobId] }]);
+                            expect(callArgs[0]).toEqual([
+                                actionRequest,
+                                { jobIdList: [`${jobId}-v2`] },
+                            ]);
+                        });
+                        it(`can ${action} a retried job in status ${status}`, function () {
```

Comment:
extra tests for cancel / retry of a retried job to make sure that if it is a retry, the retry parent ID is sent to ee2